src/components/button-group/styles.ts

```diff
@@ -0,0 +1,19 @@
+// import { css } from '@linaria/core'
+import { isMobile } from '@/styles/media'
+import { styled } from '@linaria/react'
+
+export const ElButtonGroup = styled.div`
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-2);
+  flex-wrap: wrap;
+
```

Comment:
**question:** Where do these mobile-specific CSS properties come from?

Reply:
I added that to support mobile-first CSS, as it wasn't provided in the Figma design. My aim was to implement a 100% width button approach.

Reply:
**issue:** similar to my comments above, I don't think we should be adding styles that aren't documented in Figma.

**suggestion:** Again, if you think this kind of treatment on mobile devices is important, ask Design via Figma.